azure-communication-ui/calling/src/main/res/values/azure_communication_ui_calling_strings.xml

```diff
@@ -94,7 +94,7 @@
     <!--    PopUp warning-->
     <string name="azure_communication_ui_calling_snack_bar_button_dismiss">Dismiss</string>
     <string name="azure_communication_ui_calling_alert_title">Alert</string>
-    <string name="azure_communication_ui_calling_call_state_error_call_end">You were removed from the call due to an error.</string>
+    <string name="azure_communication_ui_calling_call_state_error_call_end">You may have been removed from the meeting, or lost your connection. Try to join again.</string>
```

Comment:
I think this will cause test cases to fail. we have network test. + localization changes required. 

Please also check how the toast message looks with large text.

Reply:
Updated the screen shot in the description. 
Ran Tests and it all passed.